python_legacy/iceberg/api/expressions/expression_parser.py

```diff
@@ -126,6 +126,15 @@ def get_expr_tree(tokens):
                  binary_tuples]}
 
 
+def handle_str_expr(node):
+    if node.startswith("'") and node.endswith("'"):
```

Comment:
Will it handle some complicated cases, e.g. `'foo\'bar\'' case as well?
wondering if we can use some functions in pyparsing quotedString to solve it?